src/shared/services/toast-service.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FunctionComponent } from 'react';
+import { toast, ToastOptions } from 'react-toastify';
+
+import { Alert } from '@viaa/avo2-components';
+import { isNil, isString } from 'lodash-es';
+
+export enum TOAST_TYPE {
+	DANGER = 'danger',
+	INFO = 'info',
+	Spinner = 'spinner',
+	SUCCESS = 'success',
+}
+
+interface ToastService {
+	m: string;
+	t?: TOAST_TYPE;
+	dark?: boolean;
+	options?: ToastOptions;
+}
+
+const Toast: FunctionComponent<any> = ({ closeToast, ...rest }) => (
+	<Alert {...rest} className="u-spacer-top" close={closeToast} />
+);
+
+export default function toastService(message: ToastService | string) {
```

Comment:
mss nog een 2de optionele optie voor het type
en dan de m en t hernoemen naar message en type
dan kunnen we de meeste toast calls als volgt schrijven zonder de curleys en de m en t
```
toastService('my message', TOAST_TYPE.ERROR);
```

Reply:
Fixed